src/components/Footer/Footer.js

```diff
@@ -69,24 +76,30 @@ const Footer = () => (
 
         <ul className={styles['footer__list']}>
           <li className={styles['footer__list__first__row']}>
-            <Link to='/#' className={styles['footer__list__link']}>
+            <a
+              href='mailto:contact@hoffmann.ai'
+              className={styles['footer__list__link']}
+            >
               <Icon
                 className={styles['footer__list__icon']}
                 name='email'
                 icon={getIcon('email')}
               />
               contact@hoffmann.ai
-            </Link>
+            </a>
           </li>
           <li className={styles['footer__list__first__row']}>
-            <Link to='/#' className={styles['footer__list__link']}>
+            <a
+              href='tel:06 21 21 82 2'
```

Comment:
Did you test this? What is the caller is in another country? Should have the country code.